recipe/meta.yaml

```diff
@@ -1,17 +1,14 @@
 {% set version = "2.38.0" %}
-{% set win_version = version[:-2] %}
 
 package:
   name: graphviz
   version: {{ version }}
 
 source:
-  fn:  graphviz-{{ version }}.tar.gz  # [unix]
-  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz  # [unix]
+  url: http://pkgs.fedoraproject.org/repo/pkgs/graphviz/graphviz-2.38.0.tar.gz/5b6a829b2ac94efcd5fa3c223ed6d3ae/graphviz-2.38.0.tar.gz  # [unix]
   md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae  # [unix]
 
-  fn:  graphviz-{{ win_version }}.zip  # [win]
-  url: http://www.graphviz.org/pub/graphviz/stable/windows/graphviz-{{ win_version }}.zip  # [win]
+  url: https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.zip  # [win]
```

Comment:
Would be nice to template the URLs to make them a little nicer for use in the future.